WordPress/Tests/WP/DeprecatedParametersUnitTest.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+// All will give an ERROR.
+
+get_last_updated( 'deprecated' );
+xfn_check( '', '', 'deprecated' );
+convert_chars( '', 'deprecated' );
+comments_link( 'deprecated', 'deprecated' );
+comments_number( '', '', '', 'deprecated' );
+the_author( 'deprecated', 'deprecated' );
+wp_upload_bits( '', 'deprecated' );
+get_the_author( 'deprecated' );
+the_author_posts_link( 'deprecated' );
+add_option( '', '', 'deprecated' );
+get_user_option( '', '', 'deprecated' );
+the_attachment_link( '', '', 'deprecated' );
+trackback_rdf( 'deprecated' );
+trackback_url( 'deprecated' );
+wp_install( '', '', '', '', 'deprecated' );
+discover_pingback_server_uri( '', 'deprecated' );
+load_plugin_textdomain( '', 'deprecated' );
+wp_get_http_headers( '', 'deprecated' );
+safecss_filter_attr( '', 'deprecated' );
+wp_get_sidebars_widgets( 'deprecated' );
+get_delete_post_link( '', 'deprecated' );
+is_email( '', 'deprecated' );
+is_email( '', 'false' ); // False as a string not bool.
+update_user_status( '', '', '', 'deprecated' );
+update_blog_option( '', '', '', 'deprecated' );
+wp_notify_postauthor( '', 'deprecated' );
+wp_notify_postauthor( '', 'null' );
+wp_new_user_notification( '', 'deprecated' );
+get_wp_title_rss( 'deprecated' );
+wp_title_rss( 'deprecated' );
+
+// All will give an WARNING as they have been deprecated after WP 4.5.
+
+unregister_setting( '', '', '', 'deprecated' );
+get_category_parents( '', '', '', '', array( 'deprecated') );
+
+// All will give an WARNING as they have a dynamic variable.
```

Comment:
For practical purposes, I would like to suggest that the tests from this line to the end, be moved to the top of the test file as these tests won't change.

The version based ones above, will then be below them which will make it easier to add new tests for newly deprecated parameters + to move tests up from the warning list to the error list when a new WP version has been released without having to make drastic changes to the unit test line number arrays.